changelog.d/20240125_122858_shadinaif_FC_0012_OEP_58.md

```diff
@@ -0,0 +1 @@
+💥[Feature] Add support to `atlas pull` - FC-0012 project, OEP-58. (by @shadinaif) -->
```

Comment:
@shadinaif I found few typos after merging this pull request. Please fix them if needed.

```suggestion
- [Feature] Add support to `atlas pull` - FC-0012 project, OEP-58. (by @shadinaif)
```

cc: @Faraz32123